src/Uno.UI.RuntimeTests/Tests/Windows_UI_Xaml_Controls/Given_Border.cs

```diff
@@ -102,5 +110,193 @@ public async Task When_Border_Centered_With_Margin_Inside_Wide_Rectangle()
 			const double ScrollViewerWidth = ContentWidth + 2 * ContentMargin;
 			await WindowHelper.WaitForEqual(ScrollViewerWidth, () => SUT.ActualWidth);
 		}
+
+		[TestMethod]
+		public async Task When_CornerRadius_Border_Basic()
+		{
+			// Verify that border is drawn with the same thickness with/without CornerRadius
+			const string white = "#FFFFFFFF";
+			const string red = "#FFFF0000";
+
+#if __MACOS__
+			Assert.Inconclusive(); // MACOS interpret colors differently
+#endif
+			if (!ApiInformation.IsTypePresent("Windows.UI.Xaml.Media.Imaging.RenderTargetBitmap"))
+			{
+				Assert.Inconclusive(); // System.NotImplementedException: RenderTargetBitmap is not supported on this platform.;
+			}
+
+			var SUT = new Border_CornerRadius();
+			var sample = SUT.Sample1;
+			float eighth = (float)sample.Width / 8;
+			var result = await TakeScreenshot(SUT);
+
+			ImageAssert.HasColorAtChild(result, sample, eighth, (float)sample.Height - eighth, white, tolerance: 25);
```

Comment:
Does the original ImageAssert not work? Maybe we should be porting it properly so the tests code does not differ too much from the original?

Reply:
I think it is better now. I created the Helper Class ChildPosition to keep the code light. The same tests are used in both sample App and Runtime